07-routing-gossip.md

```diff
@@ -57,6 +57,15 @@ It is constructed as follows:
   2. the next 3 bytes: indicating the transaction index within the block
   3. the least significant 2 bytes: indicating the output index that pays to the channel.
 
+The standard human readable format for `short_channel_id` is created
+by printing the above components, in the order:
+block height, transaction index, and output index.
+Each component is printed as a decimal number,
+and separated from each other by the small letter `x`.
+For example, a `short_channel_id` might be written as `539268x845x1`,
+indicating a channel on the output 1 of the transaction at index 845
+of the block at height 539268.
```

Comment:
I'm not a native speaker, but should it be "index 845 *in* the block?

Reply:
Construction "x of y" can mean "x is a part of y" or "x comes from y" or "x is owned by y", by my understanding of this language.  So we can consider "the transaction at index N of the block" as the transaction (which happens to be at index N) that is part of the block.